### PR TITLE
cpdp-589: bpr retrieved using sautr when available for user with cl<200

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/StartController.scala [hunk 1]
```diff
@@ -118,21 +118,30 @@ class StartController @Inject()(
   }
 
 
-  private def handleInsufficientConfidenceLevel(maybeNino: Option[NINO], name: Name, maybeEmail: Option[Email])(
+  private def handleInsufficientConfidenceLevel(maybeNino: Option[NINO],
+                                                maybeSautr: Option[SAUTR],
+                                                name: Name,
+                                                maybeEmail: Option[Email])(
     implicit request: RequestWithSessionDataAndRetrievedData[AnyContent]
```
Comment (co-category) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/StartController.scala]: formatting

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/UserType.scala [hunk 2]
```diff
@@ -33,6 +33,7 @@ object UserType {
 
   final case class IndividualWithInsufficientConfidenceLevel(
                                                               nino: Option[NINO],
+                                                              sautr: Option[SAUTR],
                                                               name: Name,
                                                               email: Option[Email]
                                                             ) extends UserType
```
Comment (co-category) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/UserType.scala]: formatting

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/UserType.scala]: same here

Comment (co-category) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/UserType.scala]: The scalafmt config should force this class to be formatted to something like:

```scala
@Singleton
class BusinessPartnerRecordConnectorImpl @Inject()(
  http: HttpClient,
  val config: ServicesConfig
)(implicit ec: ExecutionContext)
```

ie the class variables are two spaces indented from the very left.

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/UserType.scala]: oh - not sure why scalafmt is not doing it automatically. Unless we're missing a setting or something. I'll change it

**Pull request merged**